Extension/src/LanguageServer/client.ts

```diff
@@ -1617,7 +1627,9 @@ export class DefaultClient implements Client {
         const initializationOptions: InitializationOptions = {
             packageVersion: util.packageJson.version,
             extensionPath: util.extensionPath,
-            storagePath: this.storagePath,
+            storagePath: this.databaseStoragePath,
```

Comment:
Is `storagePath` supposed to be `databaseStoragePath` so that it corresponds to the object received on the language server side?

Reply:
Thank you for catching this! This was causing a bug I couldn't figure out